bin/sao

```diff
@@ -116,18 +118,23 @@ defaultCommand
     type: 'boolean'
   })
   .option('templates', {
+    alias: 'l',
     desc: 'Show installed templates',
     type: 'boolean'
   })
   .option('force-npm', {
-    desc: 'Always use npm^5 instead of yarn to install templates and dependencies'
+    alias: 'f',
```

Comment:
We may preserve this flag for future use

Reply:
`npm` should be fine 🤔

Reply:
Hm... I thought one letter flags are only supported... :joycat:

Reply:
yeah you'll need `--npm` then